### PR TITLE
eq-224 Contact Header

### PR DESCRIPTION
**Pull request by hamishtaplin**
## Screenshot

screenshot 2016-04-07 08 11 28
## Changes
- Updated contact header to latest design
## Test
- `npm run compile`
- Check header matches screenshot

### CODE REVIEW

File: app/templates/layouts/_survey.html [hunk 1]
```diff
@@ -23,13 +23,22 @@ <h1 class="bar__title">Monthly Business Survey - Retail Sales Index</h1>
       </div>
 
     {% if data %}
+      {% set info = {
+        'Code': data.survey_code,
+        'Period': data.period_str,
```
Comment (hamishtaplin) [inline app/templates/layouts/_survey.html]: For some reason `data.period_str` only pulls through on the landing page.

Comment (weapdiv-david) [inline app/templates/layouts/_survey.html]: fixed in PR #132

**Comment by hamishtaplin** [comment]
This is ready for merge now.

File: app/templates/layouts/_survey.html [hunk 2]
```diff
@@ -24,12 +24,18 @@ <h1 class="bar__title">Monthly Business Survey - Retail Sales Index</h1>
...
-          <li class="info__item info__item--last">Reference: {{ data.respondent_id or '' }}</li>
-        </ul>
+        <h2 class="info__title">Survey Help</h2>
+        <div class="info__tel">
+          0300 1234 931
+        </div>
+        <dl class="info__list">
+          <dt class="info__dt">ID:</dt>
+          <dd class="info__dd">{{ data.survey_code}}</dd>
+          <dt class="info__dt">Period:</dt>
```
Comment (warrenbailey) [inline app/templates/layouts/_survey.html:34]: You've removed the checks on these data values? This means it'll error if that piece of data is missing. Is that what you wanted to do?

Comment (hamishtaplin) [inline app/templates/layouts/_survey.html:34]: We can't have empty values, under what circumstances would they fail?

Comment (warrenbailey) [inline app/templates/layouts/_survey.html:34]: They come directly from RRM, we don't check them, so technically they could be missing, depends on how defensive we want to be.

Comment (warrenbailey) [inline app/templates/layouts/_survey.html:34]: Actually checking the code, those values are required so the RRM has to provide them or the token gets rejected.

**Comment by warrenbailey** [comment]
LGTM :+1:

**Pull request merged**